src/Sarif.Viewer.VisualStudio.UnitTests/ErrorList/SarifFileWithContentsTests.cs

```diff
@@ -187,20 +187,21 @@ public SarifFileWithContentsTests()
             };
 
             TestUtilities.InitializeTestEnvironment(testLog);
+            RunIndex = CodeAnalysisResultManager.Instance.CurrentRunIndex;
```

Comment:
Comment that CurrentRunIndex is internal in CodeAnalysisResultsManager only for this test.